Yank/multistate/paralleltempering.py

```diff
@@ -226,7 +226,7 @@ class ParallelTemperingAnalyzer(ReplicaExchangeAnalyzer):
     See Also
     --------
     ReplicaExchangeAnalyzer
-    PhaseAnalyzer
+    MultiStateSamplerAnalyzer
```

Comment:
Did you mean to remove `PhaseAnalyzer` as well? I don't think the docs fill in all the methods inherited from the parent class(es), so I wanted to leave `PhaseAnalyzer` in place to so people could jump to it.

If you dont like the idea of documented ABC's, we can try to think of something better for the docs.

Reply:
Fixed!